components/Tabs2/src/MDCTabBarView.m

```diff
@@ -16,4 +16,49 @@
 
 @implementation MDCTabBarView
 
+#pragma mark - Initialization
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    self.items = @[];
+  }
+  return self;
+}
+
+#pragma mark - Properties
+
+- (void)setItems:(NSArray<UITabBarItem *> *)items {
+  NSParameterAssert(items);
+
+  if (_items != items && ![_items isEqual:items]) {
+    _items = [items copy];
+
+    // Determine new selected item, defaulting to nil.
+    UITabBarItem *newSelectedItem = nil;
+    if (_selectedItem && [_items containsObject:_selectedItem]) {
+      // Previously-selected item still around: Preserve selection.
+      newSelectedItem = _selectedItem;
+    }
+
+    _selectedItem = newSelectedItem;
+  }
+}
+
+- (void)setSelectedItem:(UITabBarItem *)selectedItem {
+  if (_selectedItem == selectedItem) {
```

Comment:
Please use the property getters and setters here except assigning `_selectedItem` since this is the setter.

Reply:
Fix applied.